c/rust/Makefile

```diff
@@ -14,6 +16,16 @@ build:
 	    ;; \
 	esac
 
+libvirtblocks_c_rust.la: ../libtool.la.in rustbuild
```

Comment:
So now we're going to have to link twice when building libvirt against Rust too... I guess I can live with that, but we should really figure out a better way.

Reply:
Well the only possible way to do this, that I can think of, is moving all the Makefile bits into `build.rs`. I'll create an issue for that so that we can track it. I also want to make it better, that will give us one more point for Rust then ;)